shell/common/shell.cc

```diff
@@ -978,9 +979,9 @@ void Shell::OnPlatformViewSetViewportMetrics(const ViewportMetrics& metrics) {
       });
 
   task_runners_.GetUITaskRunner()->PostTask(
-      [engine = engine_->GetWeakPtr(), metrics]() {
+      [engine = engine_->GetWeakPtr(), metrics, view_id]() {
```

Comment:
Consider swapping captures to match argument order:

```suggestion
      [engine = engine_->GetWeakPtr(), view_id, metrics]() {
```

I don't feel strongly about this.